ui/src/components/layout/navbar/Navbar.tsx

```diff
@@ -15,7 +15,7 @@ const Navbar = async () => {
                 <div className="flex justify-between">
                     <Link href="/" className="lg:inline hidden">
                         <Image 
-                            src="uhgroupings/uh-groupings-logo.svg" 
+                            src="uhgroupings/uh-groupings-logo.svg"
```

Comment:
Nice catch here, I was wondering why the other pages icons were not loading. Could you also make this change for the Image below this? That image is the logo on the Navbar on mobile. Thanks!

Reply:
You can disregard this, I also made the fix in another PR that was recently merged.